lbry/lbry/dht/protocol/routing_table.py

```diff
@@ -317,3 +328,6 @@ def buckets_with_contacts(self) -> int:
             if len(bucket):
                 count += 1
         return count
+
+    def get_nonempty_bucket_ranges(self) -> typing.List[typing.Tuple[int, int]]:
```

Comment:
what's this for?

Reply:
This was again needed for the buckets, sorry...